about.html

```diff
@@ -0,0 +1,143 @@
+            <li class="folow"><a href="#">Follow us</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+
+    <div class="nav-mobile">
+      <div class="cross">
+        <img src="./Images/Nav/cross.svg" alt="cross" />
+      </div>
+      <ul class="container">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#info">About</a></li>
+        <li><a href="#events">Events</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <main>
+    <section id="info" class="info">
+      <div class="data">
+        <div class="para">
+          <h1 class="jeblow">A studio that provides amazing photography</h1>
+          <div class="description">
+            <p>
+              Adam Centre Studio was started by Abdimulhin Adan. <br> Main Ofices are at Nairobi, Kenya.
+              <br> Tel: +254759935007 <br>
+              email: adamabdimulhi.001@gmail.com <br>
+              Lunga-lunga Road
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="logo-details">
+        <h1>Adam Centre Studio</h1>
+        <hr>
+      </div>
+      <div class="footer-logo">
+        <img src="./Images/Nav/picha-removebg-preview.png" alt="logo">
+      </div>
+
+    </section>
+
+    <section id="#events" class="events">
+      <div class="past">
+        <h3>See Our past global events</h3>
+        <hr>
+        <div class="event-para">
+          <p>Take a look at our past launch events that took place in KICC, Nairobi and Mombasa.</p>
+        </div>
+      </div>
+
+      <div class="event-photos">
+        <div class="photo1">
+          <img src="./Images/Nav/event1.avif" alt="event1">
+          <div class="photo-text">
+            <h5>2020</h5>
+            <p>Launched our products at KICC, Nairobi</p>
+          </div>
+        </div>
+
+        <div class="photo2">
+          <img src="./Images/Nav/event2.avif" alt="">
+          <div class="photo-text">
+            <H5>2022</H5>
+            <p>Giving credit to our customers at Mombasa</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <section id="contact"  class="patners">
+    <div class="mates">
+      <div class="all-patners">
+        <h2>Get In touch</h2>
+        <hr>
+      </div>
+      <div class="patner-icons">
+        <ul class="iconss">
+          <li class="icon-items"><a href="#"><img src="./Images/mozilla 1.png" alt="mozilla"></a></li>
+          <li class="icon-items"><a href="#"><img src="./Images/googlekorea 1.png" alt="googlekorea"></a></li>
+          <li class="icon-items"><a href="#"><img src="./Images/naver 1.png" alt="naver"></a></li>
+          <li class="icon-items"><a href="#"><img src="./Images/daumkakaologo 1.png" alt="daumkakaologo"></a></li>
+          <li class="icon-items"><a href="#"><img src="./Images/airbnb 1.png" alt="airbnb"></a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] The partner section is missing in the about page, kindly fix this. I would advise you to use `display: flex;`, `align-items: center:`, `justify-content: space-around` and `flex-wrap: wrap;`  to achieve nicely palced logos.

![image](https://user-images.githubusercontent.com/81932478/198342647-8cf507b0-a008-454c-a3f8-a3c720690994.png)

Reply:
Hello @Munsa1 The project you have just reviewed is not mine, I think you have just confused with another.